library/src/org/wordpress/passcodelock/DefaultAppLock.java

```diff
@@ -92,6 +92,19 @@ public boolean setPassword(String password) {
         return true;
     }
 
+    @Override
+    public boolean isFingerprintEnabled() {
+        return mSharedPreferences.getBoolean(BuildConfig.FINGERPRINT_ENABLED, true);
```

Comment:
Would you mind to rename this to `FINGERPRINT_ENABLED_KEY` and add the entry in `gradle.properties-example`?